llvm-bazel/llvm-project-overlay/clang/BUILD

```diff
@@ -1025,6 +1025,20 @@ cc_library(
     ],
 )
 
+gentbl(
+    name = "tooling_syntax_nodes_gen",
```

Comment:
Nit: I'd consider tooling_syntax_gen, there's likely to be at least one more (roles) that isn't specifically nodes.

Reply:
Fair enough. Note that an identical patch was already cl/342233973, so you might want to suggest that there as well :-)

Reply:
(I did, but the person doing that integrate wasn't one of the people on duty so didn't see my suggestion)

Reply:
Mailed you a CL to update it internally as well :-)